controlplane/control/digest.go

```diff
@@ -78,20 +70,12 @@ type Digest struct {
 	Value DigestValue
 }
 
-func (d Digest) CLIInfo() string {
-	var t string
-	switch d.Type {
-	case DigestTypeSt:
-		t = fmt.Sprintf("Type: Structure, %s", d.St.CLIInfo())
-	case DigestTypeValue:
-		t = fmt.Sprintf("Type: Value, %s", d.St.CLIInfo())
-	default:
-		t = "Type: Unknown"
-	}
+func (d Digest) GetUID() SamplerDigestUID {
```

Comment:
I don't think we need getters if the field is exported

Reply:
UID getters are not needed (forgot to remove them, I will do it right now). But Name getters are needed in:
```go
type named interface {
	GetName() string
}

func getEntryByName[U comparable, T named](entries map[U]T, name string) (T, bool) {
	var entry T
	var ok bool
	for _, entry = range entries {
		if entry.GetName() == name {
			ok = true
			break
		}
	}
	return entry, ok
}
```